docs/install.rst

```diff
@@ -80,3 +132,47 @@ You can check the installation by running the following test script.
   ray start --head
   python3 tests/test_install.py
 
+
+Troubleshooting
+---------------
+
+Using Alpa on Slurm
+###################
+Since Alpa relies on Ray to manage the cluster nodes, Alpa can run on a Slurm cluster as long as Ray can run on it.
+If you have trouble running Alpa on a Slurm cluster, we recommend to follow `this guide <https://docs.ray.io/en/latest/cluster/slurm.html>`__ to setup Ray on Slurm and make sure simple Ray examples
+can run without any problem, then move forward to install and run Alpa in the same environment.
+
+Common issues of running Alpa on Slurm include:
+
+- The Slurm cluster has installed additional networking proxies, so XLA client connections time out. Example errors can be found in `this thread <https://github.com/alpa-projects/alpa/issues/452#issuecomment-1134260817>`_.
+  The slurm cluster users might need to check and fix those proxies on their slurm cluster and make sure processes spawned by Alpa can see each other.
+
+- When launching a Slurm job using ``SRUN``, the users do not request enough CPU threads or GPU resources for Ray to spawn many actors on Slurm.
+  The users need to adjust the value for the argument ``--cpus-per-task`` passed to ``SRUN`` when launching Alpa. See `Slurm documentation <https://slurm.schedmd.com/srun.html>`_ for more information.
+
+You might also find the discussion under `Issue #452 <https://github.com/alpa-projects/alpa/issues/452>`__ helpful.
+
+Jaxlib, Jax, FLAX Version Problems
+##################################
+Alpa is compatible with the following Jaxlib, Jax, and Flax versions:
+- Jax==0.3.5
+- Flax==0.4.1
+- Alpa-modified Jaxlib distributed at `self-hosted PyPI <http://169.229.48.123:8080/simple/>`_ or compiled from source.
+
+However, sometimes the users might have installed other versions of Jax-based neural network libraries, such as Flax or Optax in their environment, an incompatible version of
+Jaxlib or Jax will be automatically installed by pip, and the following error might appear when importing alpa:
+
+.. code:: bash
+
+  >>> import alpa
+    ......
+    RuntimeError: jaxlib version 0.3.7 is newer than and incompatible with jax version 0.3.5. Please update your jax and/or jaxlib packages
+
+Make sure your jax version is 0.3.5, Flax version is 0.4.1 by reinstalling them following:
+
+.. code:: bash
+
+  pip3 install jax==0.3.5
+  pip3 install flax==0.4.1
```

Comment:
Does this mean we have to update this webpage every time we release the package?

Reply:
no, why you think so?